tests/load/utils.py

```diff
@@ -302,6 +299,27 @@ def destinations_configs(
                 bucket_url=AZ_BUCKET,
                 extra_info="az-authorization",
             ),
+            DestinationTestConfiguration(
```

Comment:
one of these tests, it does not matter which one, need to be moved into the default_sql_configs sections, below clickouse, so we run the default sql tests against it.